documentation/asciidoc/topics/proc_connecting_cli.adoc

```diff
@@ -0,0 +1,59 @@
+[id='connecting-cli_{context}']
+= Connecting the {brandname} CLI
+Use the command line interface (CLI) to connect to your {brandname} cluster and perform administrative operations.
+
+.Prerequisites
+
+* Retrieve the necessary client connection details.
```

Comment:
Where do we explain how todo this? Can we link to it?

Reply:
It's in a topic that comes immediately before this one. I think I'm just going to remove this prereq as it's kind of wordy and just dangling in there.